resources/fetch-maven.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+if ! command -v unzip &> /dev/null; then
+    echo "$0 requires unzip"
+fi;
+
+VER=3.3.0
+PROJECT_DIR="$(realpath "$(dirname "$0")/../")"
```

Comment:
Yes, I noticed that in libjitsi after this PR and used `"$(cd "$(dirname "$0")"; pwd -P)/../"` instead. The script itself is a convenience to download the binaries from the latest release, so it's not strictly necessary. And this particular line is also just a convenience so that the script can be called from anywhere (and not just from within that directory).

Reply:
If your goal is to suppress the `/../`, you have the order of your parentheses wrong -- I'll push a fix.

Reply:
The goal is just to get the directory above where the script lies. It works both ways, but yours is cleaner.